modules/instrument_builder/jsx/react.instrument_builder.js

```diff
@@ -264,23 +284,21 @@ var DisplayElements = React.createClass({
               <LorisElement element={element}/>
             </td>
             <td style={colStyles}>
-              <button onClick={this.props.editElement.bind(this, i)} className="button">
+              <button onClick={this.props.editElement.bind(null, i)} className="button">
```

Comment:
React shows this warning when passing "this" 
![image](https://cloud.githubusercontent.com/assets/6627543/21936976/4ee4c008-d983-11e6-93eb-2537820f8b8d.png)

I believe it is because in that scope, this does not refer to React component anymore.
In any case, the argument is not used by the editElement function